src/controllers/tests/diff.controller.test.ts

```diff
@@ -0,0 +1,18 @@
+import request from 'supertest';
+import { App } from '../../app';
+
+import { DiffController } from '../../controllers/diff.controller';
+
+describe('DiffController', () => {
+  describe('[POST] /diff', () => {
+    it('should diff files', async () => {
+      const app = new App([new DiffController()]);
+      return request(app.getServer())
+        .post('/v1/diff')
+        .send({
+          asyncapis: ['./diff/asyncapi-diff-test-1.yaml','./diff/asyncapi-diff-test-2.yaml']
```

Comment:
You operate on content sent by the user, you cannot use filesystem because these files are sent as a value anyway, not a file path. So you need to load that files before making request and then pass them as value. Did I make it clearer?

Reply:
I am not able to get this part. How should I pass the file like using fs or is there any other way?